### PR TITLE
Add timelines example scripts

### PR DESCRIPTION
**Pull request by mr-devs**
Changes:
- add `examples/tweet_timeline.py` to pull the most recent tweets sent by a single user ID
- add `examples/mentions_timeline.py` to pull the most recent mentions of a single user ID
- update `examples/README.md` chart with new examples and some language for clarity

### CODE REVIEW

File: examples/mentions_timeline.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python3
+
+
+# Script Information
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+"""
+PURPOSE:
+    - Script to scrape up to the most recent 800 tweets sent by a provided
+    user ID by utilizing the V2 Twitter mentions timeline endpoint.
```
Comment (yangkcatiu) [inline examples/mentions_timeline.py:9]: We can maybe use the word from Twitter's doc "Returns Tweets mentioning a single user specified by the requested user ID." Right now it sounds like this endpoint will return the tweets from the focal user.

Comment (mr-devs) [reply examples/mentions_timeline.py:9]: Good call, I will update.

Also, thanks for being all over these PRs! 💪

Comment (yangkcatiu) [reply examples/mentions_timeline.py:9]: No prob!

**Comment by yangkcatiu** [comment]
This PR looks great to me! Please merge once the docstring is fixed.

**Pull request merged**